tempesta_fw/Makefile

```diff
@@ -24,8 +24,9 @@ endif
 
 obj-m	= tempesta_fw.o
 tempesta_fw-objs = addr.o cache.o classifier.o client.o connection.o debugfs.o \
-		   filter.o gfsm.o  hash.o http.o http_match.o http_msg.o \
-		   http_parser.o if.o main.o pool.o sched.o server.o session.o \
+		   cfg_node.o cfg_mod.o cfg_spec.o cfg_parser.o cfg_userspace_if.o \
```

Comment:
Now we have only

        tempesta/tempesta_fw $ ls cfg_*
        cfg_mod.c  cfg_mod.h  cfg_parser.c  cfg_parser.h